src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -143,7 +145,21 @@ public FeedbackSessionAttributes copyFeedbackSession(String newFeedbackSessionNa
             throw new EntityDoesNotExistException(
                     "Trying to get feedback sessions for a course that does not exist.");
         }
+        return getFeedbackSessionsForUserInCourseSkipCheck(courseId, userEmail);
+    }
 
+    /**
+     * Under the reasonable assumption (or after verifying) that the course <b>courseId</b>
+     * exists, this method gets the feedback sessions for the specified user in the course.
```

Comment:
This comment is not accurate. The method does not assume any such thing, and it certainly does not verify anything. It simply retrieves sessions without checking if the course exists.